README.md

```diff
@@ -1,15 +1,18 @@
 # aconfig
+> Simple, useful and opinionated config loader

```

Comment:
why?

Reply:
The most common style on GitHub I've seen is title, description,  badges, and then other stuff.

I'll revert though